### PR TITLE
Draws PolygonItems for octagon and note

### PR DESCRIPTION
**Pull request by naveedhd**
For adding unique shapes for Parallels, Choosers etc.

### CODE REVIEW

**Comment by naveedhd** [comment]
*TODO*: use a different shape for doubleoctagon (something single)

**Comment by naveedhd** [comment]
Using hexagon for Chooser now. (instead of double octagon)

**Comment by naveedhd** [comment]
Related PRs:

* [py_trees_msgs #3](https://github.com/stonier/py_trees_msgs/pull/3)
* rqt_py_trees #7 
* [py_trees_suite #30](https://github.com/stonier/py_trees_suite/pull/30)

**Comment by stonier** [comment]
Looks like you're using `octagon` for the selector now.  I'm guessing this is to differentiate it from a behaviour.

One result of this work should be the dot generator and rqt visualisor should be as close as possible.

chooser

selector

Oh, selector there is octagon :) I thought it was elliptic. Relevant dot generator code is [here](https://github.com/stonier/py_trees_suite/blob/devel/py_trees/src/py_trees/display.py#L126).

**Comment by naveedhd** [comment]
Drawing the double octagon using single shape looks like this:

double

**Review by stonier — APPROVED** [review]

File: src/rqt_py_trees/dotcode_behaviour.py [hunk 1]
```diff
@@ -130,6 +138,10 @@ def type_to_string(self, behaviour_type):
             return 'Sequence'
         elif behaviour_type == py_trees_msgs.Behaviour.SELECTOR:
             return 'Selector'
+        elif behaviour_type == py_trees_msgs.Behaviour.PARALLEL:
+            return 'Parallel'
+        elif behaviour_type == py_trees_msgs.Behaviour.CHOOSER:
```
Comment (stonier) [inline src/rqt_py_trees/dotcode_behaviour.py:143]: We could probably stringify the name here - python magic. Not critical though - we should have a non-increasing finite set to deal with.

**Pull request merged**